config/manifests/bases/elasticsearch-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,298 @@
+        path: nodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:nodeSelector
+      - description: The desired number of Kibana Pods for the Visualization component
+        displayName: Kibana Size
+        path: replicas
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+      - description: The resource requirements for the Kibana nodes
+        displayName: Kibana Resource Requirements
+        path: resources
+      statusDescriptors:
+      - description: The status for each of the Kibana pods for the Visualization component
+        displayName: Kibana Status
+        path: pods
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podStatuses
+      version: v1
+  description: "The Elasticsearch Operator for OCP provides a means for configuring and managing an Elasticsearch cluster for use in tracing \nand cluster logging as well as a Kibana instance to connect to it.\nThis operator only supports OCP Cluster Logging and Jaeger.  It is tightly coupled to each and is not currently capable of\nbeing used as a general purpose manager of Elasticsearch clusters running on OCP.\n\nIt is recommended that this operator be installed in the `openshift-operators-redhat` namespace to \nproperly support the Cluster Logging and Jaeger use cases.\n\nOnce installed, the operator provides the following features for **Elasticsearch**:\n* **Create/Destroy**: Deploy an Elasticsearch cluster to the same namespace in which the elasticsearch CR is created.\n* **Update**: Changes to the elasticsearch CR will be scheduled and applied to the cluster in a controlled manner (most often as a rolling upgrade).\n* **Cluster health**: The operator will periodically poll the cluster to evaluate its current health (such as the number of active shards and if any cluster nodes have reached their storage watermark usage).\n* **Redeploys**: In the case where the provided secrets are updated, the Elasticsearch Operator will schedule and perform a full cluster restart.\n* **Index management**: The Elasticsearch Operator will create cronjobs to perform index management such as roll over and deletion.\n\nOnce installed, the operator provides the following features for **Kibana**:\n* **Create/Destroy**: Deploy a Kibana instance to the same namespace in which the kibana CR is created (this should be the same namespace as the elasticsearch CR).\n* **Update**: Changes to the kibana CR will be scheduled and applied to the cluster in a controlled manner.\n* **Redeploys**: In the case where the provided secrets are updated, the Elasticsearch Operator will perform a restart.\n\n### Additionally provided features\n* Out of the box multitenancy that is integrated with OCP user access control.\n* Document Level Security\n* mTLS communication between Elasticsearch, Kibana, Index Management cronjobs, and CLO's Fluentd\n* OCP prometheus dashboard for Elasticsearch clusters\n* Prometheus Alerting rules\n"
```

Comment:
Is this a mistake here?

Reply:
I copied the description from the original file and `make bundle` auto-formats the content to this form.